src/Azure/Provider.php

```diff
@@ -63,9 +70,6 @@ public function getAccessToken($code)
     protected function getUserByToken($token)
     {
         $response = $this->getHttpClient()->get($this->graphUrl, [
-            RequestOptions::QUERY => [
-                'api-version' => $this->version,
```

Comment:
So in this case the `version` class property is not used anymore and can be droped

Reply:
it's right, ok